OptionSelector/CQCategory.h

```diff
@@ -0,0 +1,17 @@
+//
+//  CQCategory.h
+//  Glycemic Index of Common Foods OptionSelector App
+//
+//  Created by Ayuna Vogel on 8/10/15.
+//  Copyright (c) 2015 Ayuna Vogel. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CQCategory : NSObject
```

Comment:
Hi Caleb, thank you for reviewing the code! Could you please expand on this a little bit? I'd like to keep good practice but it's not very clear to me as what would be a good comment for this project. Could you please give an example specific to this project? Thank you so much!